src/tests/Common/CLRTest.Jit.targets

```diff
@@ -63,6 +63,15 @@ export RunningIlasmRoundTrip=
 if [ -z "$DoLink" -a ! -z "$RunningIlasmRoundTrip" ];
 then
   mkdir IL-RT
+  if [ "$(IsMergedTestRunnerAssembly)" = "true" ]
+  then
+    python3 $(AssemblyName)_ilasmroundtrip.py
+    if [ $ERRORLEVEL -ne 0 ]
```

Comment:
The lab script didn't fail because the value is in `$?`, not `$ERRORLEVEL`.  Other parts of the script do `ERRORLEVEL=$?` because they want to save the value for another use.